python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_defs.py

```diff
@@ -655,6 +656,37 @@ def test_op_config(
     assert all_keys == expected_keys
 
 
+@pytest.mark.parametrize("instances", [[{"target": "target_a"}, {"target": "target_b"}]])
+def test_op_custom_name(instances, test_project_dir, dbt_config_dir, dbt_target_dir):
```

Comment:
I think this test could be simplified a ton, just by having all of the load_assets_from_dbt_manifest calls share the same manifest.json file (sample_manifest.json would work).

as long as the keys are distinct (which is handled by the key_prefix already), the actual contents of the manifest shouldn't matter for the purposes of this test.

also, as there's technically only one set of instances, we can probably skip parameterizing the test and just hard-code those in.